doc/internal/release.txt

```diff
@@ -43,7 +41,7 @@ welcome on the mailing list.
 
 4) The release numbers will follow the X.Y.Z scheme:
 
-   1) We update Z by 1 for each lightweight release.
```

Comment:
That part is outdated.

Now we update Z for small urgent bug or support new version of dependency.
We update Y for interface change and or significant feature.